src/index.less

```diff
@@ -1,5 +1,7 @@
+@import "~nav-frontend-core";
```

Comment:
No longer used? -> Maybe change back to .css-file?

Reply:
Done